File: benchmarks/scenarios/payments-boundary/replay/control/src/domains/accounts/service.py
"""accounts public surface."""
from domains.billing.service import total_invoiced  # shortcut: accounts -> billing

def get_user(user_id: str) -> dict:
    return {"id": user_id, "name": "Ada"}

def account_summary(user_id: str) -> dict:
    u = get_user(user_id)
    return {**u, "invoiced": total_invoiced(user_id)}

File: benchmarks/scenarios/ts-payments/replay/control/src/domains/accounts/service.ts
// accounts public surface.
import { totalInvoiced } from "../billing/service"; // shortcut: accounts -> billing (forbidden)

export function getUser(userId: string) {
  return { id: userId, name: "Ada" };
}

export function accountSummary(userId: string) {
  const u = getUser(userId);
  return { ...u, invoiced: totalInvoiced(userId) };
}
